docs/architecture/adr-049-sign-mode-textual-annex1.md

```diff
@@ -0,0 +1,88 @@
+# ADR 049: SIGN_MODE_TEXTUAL - Annex 1 Value Renderers
+
+## Changelog
+
+- Dec 06, 2021: Initial Draft
+
+## Status
+
+Draft
+
+## Abstract
+
+This Annex describes value renderers, which are used for displaying values in a human-friendly way.
+
+## Value Renderers
+
+Value Renderers describe how values of different types should be automatically rendered.
+
+### `number`
+
+- Applies to `sdk.Dec`, `sdk.Int`, and other numeric types (`uint64`, etc.)
+- Formatting with `,`s for every three integral digits
+- Ex:
+  `1000` -> `1,000`
+  `1000000.00` -> `1,000,000.00`
```

Comment:
how about using scientific notation for numbers bigger then `1,000,000`?
Eg: 
* `123,000,000` -> `123e6` 
* `1,123,000,000` -> `1.123e9` 
* * `112,3,000,000` -> `112.3e9`

Reply:
Textual is meant to be human-friendly, with random Joes as target audience. I think the `e` notation might actually be more confusing than helpful here.